enterprise/backend/src/metabase_enterprise/sso/integrations/jwt.clj

```diff
@@ -106,7 +106,8 @@
 
 (defn ^:private generate-response-token
   [session jwt-data]
-  (validation/check-embedding-enabled)
+  (api/check (embed.settings/enable-embedding-sdk)
+             [400 (tru "SDK Embedding is not enabled.")])
```

Comment:
nit - it's not super standard but we use error 402 ("payment required") in some other spots where we're checking premium features

Reply:
good call, I'll use 402.